### ISSUE TITLE
Alt+scroll doesn't work

### ISSUE THREAD

**Issue by YohananDiamond** [yuxshao/ptcollab#110]
Hello!
I'm trying ptcollab out and everything seems to work fine, except I haven't been able to change the quantization. The related keybinding is `Alt+Scroll`, which isn't working for me for some reason. `Alt+Shift+Left/Right` (for expanding selection) does, though.

I'm running ptcollab on Manjaro, under Wayland, and that might be the issue because I had some problems running ptcollab and had to use `QT_QPA_PLATFORM=xcb` for it to work (via XWayland).

**Comment by ewancg** [comment]
Which DE? Alt+scroll was reserved for zoom when I was on XFCE

**Comment by YohananDiamond** [comment]
I'm using [river](https://github.com/riverwm/river) (not really a DE, just a compositor). I've also checked my keybindings and I haven't found anything with Alt+Scroll.

**Comment by ewancg** [comment]
My first thought is that alt when not associated with other key held is related to menu stuff (select menu bar item), and your scrolling is either being ignored or cycling through the menu bar items. I don't think I added any accessibility highlighting stuff to the custom themes, try the system thing & see if when u try to alt scroll the menu bar gets selected. If so a fix is fairly easy as we can just deny focus

**Comment by YohananDiamond** [comment]
What would be 'the system thing'?
Also, I've tested it on firefox at least and over here it works (what it does is... navigating through a tab's history, I think)

**Comment by ewancg** [comment]
Sorry I was vague, this fella
![image](https://github.com/yuxshao/ptcollab/assets/36348486/0b5d1238-4b62-4cf5-bf01-c934bb3d4e4e)

I also didn't do a very good job describing what I think is happening. Qt & your window manager in tandem are expecting Alt to be used for key shortcuts, but I think that using Alt with mouse events might not be treated the same way. Since Alt is usually used to give focus to the menu bar items for keyboard navigation, maybe something's happening there. If there would be anything different that your setup has from known working ones, it would be the window manager.

**Comment by YohananDiamond** [comment]
I did some more testing here and I think I found the issue! I previously stated I was using `QT_QPA_PLATFORM=xcb`, but I just noticed I wasn't (I did that only once). Setting that env var solved the issue.
I think Qt on Wayland still has some rough edges ([there's been some problems in LMMS related to this as well](https://github.com/LMMS/lmms/issues/6291), although for drag and drop instead of alt+scroll).
Thanks for the help, anyway!
